packages/plugins/override-fields/README.md

```diff
@@ -0,0 +1,50 @@
+## `@envelop/override-fields`
+
+This plugin allows you to target specific fields and override their default behavior.  
+This is useful when you want to control parts of your schema, f.i. if they are sensible for your business or incur an extra cost from underlying data sources.  
+With this plugin, you can define conditions to replace the resolver functions of your targeted fields to return arbitrary values.
+
+## Getting Started
+
+```
+yarn add @envelop/override-fields
+```
+
+## Basic Usage
+
+You can target fields either by referencing them as plain strings or by using a Regular Expression.  
+You can access subfields with a "dot" notation, as you would do when accessing properties of an object.
+
+```ts
+import { envelop } from '@envelop/core';
+import { useOverrideFields } from '@envelop/override-fields';
+
+const getEnveloped = envelop({
+  plugins: [
+    useOverrideFields({
+      fields: ['continents', 'country.callingCodes.countries', /country\.aplhaCode.*/],
```

Comment:
I think this might need to be `Type.Field` instead of just field name? Otherwise you might effect types that you don't want to?

Reply:
Yes, you're right.
It is something I did think about during development, but then I forgot to implement.

Will add that